workers.tf

```diff
@@ -131,6 +131,12 @@ resource "aws_iam_role_policy_attachment" "workers_AmazonEC2ContainerRegistryRea
   role       = aws_iam_role.workers[0].name
 }
 
+resource "aws_iam_role_policy_attachment" "workers_AmazonSSMManagedInstanceCore" {
+  count      = var.manage_worker_iam_resources && var.create_eks ? 1 : 0
```

Comment:
will this update worker roles on an an existing cluster?

Reply:
@jweaverBW   yes it will update existing role